src/ring_theory/power_series/basic.lean

```diff
@@ -1284,6 +1284,18 @@ by simp
 by { ext, simp only [ring_hom.id_apply, rescale, one_pow, coeff_mk, one_mul,
   ring_hom.coe_mk], }
 
+lemma rescale_mk (f : ℕ → R) (a : R) :
+  rescale a (mk f) = mk (λ n : ℕ, a^n * (f n)) :=
+by { ext, rw [coeff_rescale, coeff_mk, coeff_mk], }
+
+lemma rescale_comp_eq_mul (f : power_series R) (a b : R) :
+  rescale b (rescale a f) = rescale (a * b) f :=
```

Comment:
I think your lemma should be called `rescale_rescale`, but we should also have the following:
```lean
lemma rescale_comp_eq_mul (a b : R) :
  rescale (a * b) = (rescale b).comp (rescale a) :=
```

Reply:
Done!